packages/mdc-list/README.md

```diff
@@ -418,6 +418,9 @@ Use `role="listbox"` only for single selection list, without this role the `ul`
 Do not use `aria-orientation` attribute for standard list (i.e., `role="list"`), use component's `vertical` property to set the orientation
 to vertical.
 
+Single selection list supports `aria-selected` and `aria-current` attributes. List automatically detects the presence of these attributes
```

Comment:
The "Pre-selected list item" and "Setup in `singleSelection()`" sections still mention `aria-selected` exclusively... should their wording be broadened to mention `aria-current`? (Maybe not in the single-selection case if that's expected to only ever apply for roles where `aria-selected` is valid?)

Reply:
I modified "Pre-selected list item" and "Setup in singleSelection()" section cross-linking to "Accessibility" section.